src/pages/index.tsx

```diff
@@ -12,8 +49,97 @@ export default function Home() {
         <meta name="description" content="Website for Init" />
       </Head>
 
+      {/* Navbar component */}
       {/* Main content of the Home component */}
-      <main className="flex min-h-screen flex-col bg-primary"></main>
+      <main className="flex min-h-screen flex-col bg-primary">
+        <div className="hero-wrapper-2 flex flex-col items-center justify-center sm:flex-row sm:items-start sm:pt-10 md:flex-row">
+          <div className="pr-0 text-center sm:pr-40 sm:text-left">
+            <h1 className="pb-30 text-9xl">INIT</h1>
+            <p className="pt-10 text-3xl">
+              South Florida&apos;s Largest <br />
+              Tech Community
+            </p>
+            <div className="flex flex-row space-x-5 pl-10 pt-10 sm:pl-0">
+              <button
+                type="button"
+                className="rounded-lg  bg-yellow-300 px-6 py-3.5 text-center text-base font-medium text-black hover:bg-blue-800 focus:outline-none focus:ring-4 focus:ring-blue-300 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800"
+              >
+                Join Now
+              </button>
+              <button
+                type="button"
+                className="rounded-lg bg-yellow-300 px-6 py-3.5 text-center text-base font-medium text-black hover:bg-blue-800 focus:outline-none focus:ring-4 focus:ring-blue-300 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800"
+              >
+                Extra large
+              </button>
```

Comment:
I would change this to something more descriptive. https://weareinit.org/ is a good example. Could probably use that exact paragraph if you would like. Also, the "extra large" button could probably be a link to the Discord.

Reply:
![image](https://github.com/dabarcenas6921/init-dashboard/assets/90057009/89cf61ce-88df-4dcf-b8c3-3b5e84538a23)

Also think we should avoid putting event info in this carousel and just keep it to club pictures.